pages/ksphere/kommander/latest/Architecture/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: layout.pug
+navigationTitle: Architecture
+title: Architecture
+excerpt: Kommander architecture diagram
+menuWeight: 3
+---
+
+Kommander provides management and insights for multiple Kubernetes clusters, and supports both Konvoy or non-Konvoy clusters. Kommander includes essential enterprise-grade capabilities such as:
+
+- Powerful user interface optimized to manage multiple clusters
+- Automated lifecycle management
+- Distributed configuration management
```

Comment:
This list seems incomplete especially in the context of `enterprise-grade capabilities`. Maybe include more of the features you mentioned in PR https://github.com/mesosphere/dcos-docs-site/pull/2761/files#diff-fb38da42f0b360e916c55c85a1688842R25 ?

ie:
- centralized monitoring, logging, and alerts
- governance and policy administrtation
- centralized authentication